File: 0412-fizz-buzz/0412-fizz-buzz.java
class Solution {
    public List<String> fizzBuzz(int n) {
        String []ans = new String[n];
        for(int i=0;i<n;i++)
        {
            if((i+1) % 3==0 && (i+1) % 5==0)
            ans[i]="FizzBuzz";
            else if((i+1) % 3==0)
            ans[i]="Fizz";
            else if((i+1) % 5 ==0)
            ans[i]="Buzz";
            else
            ans[i]=Integer.toString(i+1);
        }
        List<String> l=Arrays.asList(ans);
        return l;
    }
}

File: 0412-fizz-buzz/0412-fizz-buzz.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        ans=[]
        for k in range(n):
            ans.append(k+1)
        for i in range(n):
            if((i+1) % 3==0 and (i+1) % 5==0):
                ans[i]="FizzBuzz"
            elif((i+1) % 3==0):
                ans[i]="Fizz"
            elif((i+1) % 5 ==0):
                ans[i]="Buzz"
            else:
                ans[i]=str(i+1)
        return ans    
        